StripeConnect/StripeConnect/Source/Components/PayoutsViewController.swift

```diff
@@ -54,14 +54,14 @@ public protocol PayoutsViewControllerDelegate: AnyObject {
        - payouts: The payouts component that errored when loading
        - error: The error that occurred when loading the component
      */
-    func payoutsLoadDidFail(_ payouts: PayoutsViewController,
-                            withError error: Error)
+    func payouts(_ payouts: PayoutsViewController,
+                 didFailLoadWithError error: Error)
```

Comment:
We had intended to make this consistent with the AccountOnboarding delegate style, but it slipped past API review. Paper trail of API approval:
https://docs.google.com/document/d/195BaU6k2J-2CAs9anNE6e4OlZO34N-e2HTYc6pacpTw/edit?pli=1#bookmark=id.dk457iw2wtca